src/main/java/ch/njol/skript/expressions/ExprScoreboardTags.java

```diff
@@ -28,14 +28,16 @@
 		"Also you can use use the <a href='#CondHasScoreboardTag'>Has Scoreboard Tag</a> condition to check whether an entity has the given tags.",
 		"",
 		"Requires Minecraft 1.11+ (actually added in 1.9 to the game, but added in 1.11 to Spigot)."})
-@Examples({"on spawn of a monster:",
-        "\tif the spawn reason is mob spawner:",
-        "\t\tadd \"spawned by a spawner\" to the scoreboard tags of event-entity",
-        "",
-        "on death of a monster:",
-        "\tif the attacker is a player:",
-        "\t\tif the victim doesn't have the scoreboard tag \"spawned by a spawner\":",
-        "\t\t\tadd 1$ to attacker's balance"})
+@Example("""
+	on spawn of a monster:
+		if the spawn reason is mob spawner:
+			add "spawned by a spawner" to the scoreboard tags of event-entity
+
+	on death of a monster:
+		if the attacker is a player:
+			if the victim doesn't have the scoreboard tag "spawned by a spawner":
+				add 1$ to attacker's balance
+	""")
```

Comment:
these are multiple examples

Reply:
it's one cohesive example with 2 structures